browser-sign-in/OktaBrowserSignInUITests/Pages/WebLogin.swift

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright (c) 2022-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+// The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+//
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and limitations under the License.
+//
+
+import Foundation
+import XCTest
+
+protocol WebLogin {
```

Comment:
Is this test code lifted from the okta-mobile-swift test suite?  Because for the purposes of these sample apps, we might be able to streamline / simplify the tests to minimize the amount of code needed.  e.g. we may not need to make a protocol, and all that other baggage, to handle these UI tests.

Reply:
Yes, I wanted to re-use some of those code but I see your point.